apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -716,6 +719,30 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
         );
     };
 
+    /**
+     * Returns the documentation links based on the application type.
+     *
+     * @return {string}
+     */
+    const resolveDocumentationLink = (): string => {
+
+        if (selectedTemplate?.templateId === "single-page-application") {
```

Comment:
Let's move these to an enum as well. (`ApplicationTemplateTypes`)
```suggestion
        if (selectedTemplate?.templateId === ApplicationTemplateTypes.SPA) {
```

Reply:
Fixed with 249e320604fcef016aaffc5e2bb956c0cd74921f